express_server.js

```diff
@@ -6,135 +6,175 @@ const cookieSession = require("cookie-session");
 const bcrypt = require("bcrypt");
 const methodOverride = require("method-override");
 // helper functions:
-const { generateID, getUserByEmail, userURLs, copy } = require("./helpers.js");
+const {
+  areValidEmailPassword,
+  generateID,
+  getUserByEmail,
+  hashPassword,
+  userURLs,
+  emailExists,
+} = require("./helpers.js");
```

Comment:
Added some extra helpers. Always look for ways to make your code more modular. If there's logic repeating then you can create a helper function for that.